tests/test_vxlan_vtep_vni.rb

```diff
@@ -210,4 +214,23 @@ def test_suppress_arp
     vni.suppress_arp = vni.default_suppress_arp
     refute(vni.suppress_arp, 'suppress_arp should be disabled')
   end
+
+  def test_suppress_uuc
+    skip('Platform does not support suppress-unknown-unicast') unless
+    node.cmd_ref.supports?('vxlan_vtep_vni', 'suppress_uuc')
```

Comment:
You're not giving us a lot to go on here...

In any case, I pulled a copy of your fork and it works for me with these changes:

```
   def test_suppress_uuc
-    skip('Platform does not support suppress-unknown-unicast') unless
-    node.cmd_ref.supports?('vxlan_vtep_vni', 'suppress_uuc')
     vni = VxlanVtepVni.new('nve1', '6000')
+    if validate_property_excluded?('vxlan_vtep_vni', 'suppress_uuc')
+      assert_nil(vni.suppress_uuc)
+      assert_nil(vni.default_suppress_uuc)
+      assert_raises(Cisco::UnsupportedError) { vni.suppress_uuc = true }
+      return
+    end
 
     # Test: Check suppress_uuc is not configured.
```

I also made some changes to setup and teardown to make this run cleaner:

```
 class TestVxlanVtepVni < CiscoTestCase
   @skip_unless_supported = 'vxlan_vtep_vni'
+  @@pre_clean_needed = true # rubocop:disable Style/ClassVars
 
   def setup
     super
-    config('no feature nv overlay')
-    config('feature nv overlay')
-    config('interface nve1')
+    return unless @@pre_clean_needed
+
+    # Disable features on initial cleanup only as it's too expensive
+    # to do a disable for every testcase.
+    config_no_warn('no feature nv overlay')
+    config_no_warn('no feature vn-segment-vlan-based') if
+      VxlanVtep.mt_lite_support
+    config_no_warn('feature nv overlay')
+    # nv overlay is a slow start on some platforms
+    sleep 1
+    @@pre_clean_needed = false
   end
 
   def teardown
-    config('no feature nv overlay')
+    config_no_warn('no interface nve1')
     super
   end
```